Regression tester regressed 7ffd2d

[start automated regression testsuite report]

Dear all,

this is the automated regression testsuite reporting a new regression between
- new revision: 7ffd2d452c215d6a524600eac585c2e1ee71b18b  [regression testsuite results](https://cdash.dealii.org/index.php?project=deal.II&filtercount=1&showfilters=1&field1=revision&compare1=63&value1=7ffd2d)
- old revision: adbb1e237f38ec31fd2d23b5ce102ebdbe4149e6  [regression testsuite results](https://cdash.dealii.org/index.php?project=deal.II&filtercount=1&showfilters=1&field1=revision&compare1=63&value1=adbb1e)

Summary:
- :angry: new test failures introduced for configuration [GNU-13.2.0-master-ubuntu-24.10](https://cdash.dealii.org/build/1888) on site tester-tng

I have identified the following pull requests as possible candidates:
- <!-- Bump crate-ci/typos from 1.19.0 to 1.23.6 -->#17564 (authored-by: @app/dependabot; merged-by: @masterleinad)


Notes:
 - The regression reported above is a subset of the following full set of regressions compared to the baseline:
   - :disappointed: build warnings for configuration [GNU-10.2.1-master-debian-11](https://cdash.dealii.org/build/1875) on site tester-tng
   - :disappointed: build warnings for configuration [GNU-11.4.0-master-ubuntu-lts-22.04](https://cdash.dealii.org/build/1876) on site tester-tng
   - :disappointed: build warnings for configuration [GNU-12.2.0-master-debian-12](https://cdash.dealii.org/build/1887) on site tester-tng
   - :angry: test failures for configuration [GNU-12.2.0-master-debian-12](https://cdash.dealii.org/build/1887) on site tester-tng
   - :disappointed: build warnings for configuration [GNU-13.2.0-master-avx256-lts-24.04](https://cdash.dealii.org/build/1889) on site tester-tng
   - :angry: test failures for configuration [GNU-13.2.0-master-avx256-lts-24.04](https://cdash.dealii.org/build/1889) on site tester-tng
   - :disappointed: build warnings for configuration [GNU-13.2.0-master-avx512-lts-24.04](https://cdash.dealii.org/build/1891) on site tester-tng
   - :angry: test failures for configuration [GNU-13.2.0-master-avx512-lts-24.04](https://cdash.dealii.org/build/1891) on site tester-tng
   - :disappointed: build warnings for configuration [GNU-13.2.0-master-ubuntu-24.10](https://cdash.dealii.org/build/1888) on site tester-tng
   - :angry: test failures for configuration [GNU-13.2.0-master-ubuntu-24.10](https://cdash.dealii.org/build/1888) on site tester-tng
   - :disappointed: build warnings for configuration [GNU-13.2.0-master-ubuntu-lts-24.04](https://cdash.dealii.org/build/1890) on site tester-tng
   - :angry: test failures for configuration [GNU-13.2.0-master-ubuntu-lts-24.04](https://cdash.dealii.org/build/1890) on site tester-tng
   - :disappointed: build warnings for configuration [GNU-13.3.0-master-debian-13](https://cdash.dealii.org/build/1885) on site tester-tng
   - :angry: test failures for configuration [GNU-13.3.0-master-debian-13](https://cdash.dealii.org/build/1885) on site tester-tng
   - :disappointed: build warnings for configuration [GNU-9.4.0-master-ubuntu-lts-20.04](https://cdash.dealii.org/build/1877) on site tester-tng
   - :disappointed: build warnings for configuration [Clang-13.0.1-master-autodetection-20230116](https://cdash.dealii.org/build/1896) on site tester-tos
   - :disappointed: build warnings for configuration [Clang-14.0.6-master-autodetection-20230116](https://cdash.dealii.org/build/1895) on site tester-tos
   - :disappointed: build warnings for configuration [Clang-15.0.6-master-autodetection-20230116](https://cdash.dealii.org/build/1894) on site tester-tos
   - :disappointed: build warnings for configuration [Clang-16.0.6-master-autodetection-20230507](https://cdash.dealii.org/build/1893) on site tester-tos
   - :disappointed: build warnings for configuration [Clang-16.0.6-master-taskflow-20230708](https://cdash.dealii.org/build/1892) on site tester-tos
   - :disappointed: build warnings for configuration [Clang-17.0.6-master-autodetection-20240704](https://cdash.dealii.org/build/1886) on site tester-tos
   - :disappointed: build warnings for configuration [Clang-18.1.8-master-autodetection-20240704](https://cdash.dealii.org/build/1900) on site tester-tos
   - :angry: test failures for configuration [Clang-18.1.8-master-autodetection-20240704](https://cdash.dealii.org/build/1900) on site tester-tos
   - :disappointed: build warnings for configuration [Clang-18.1.8-master-unity_build-20240704](https://cdash.dealii.org/build/1883) on site tester-tos
   - :disappointed: build warnings for configuration [GNU-10.4.1-master-autodetection-20230116](https://cdash.dealii.org/build/1882) on site tester-tos
   - :disappointed: build warnings for configuration [GNU-11.3.1-master-autodetection-20230116](https://cdash.dealii.org/build/1881) on site tester-tos
   - :disappointed: build warnings for configuration [GNU-12.2.1-master-autodetection-20230116](https://cdash.dealii.org/build/1880) on site tester-tos
   - :disappointed: build warnings for configuration [GNU-13.1.1-master-64bit_indices-20230507](https://cdash.dealii.org/build/1899) on site tester-tos
   - :angry: test failures for configuration [GNU-13.1.1-master-64bit_indices-20230507](https://cdash.dealii.org/build/1899) on site tester-tos
   - :disappointed: build warnings for configuration [GNU-13.1.1-master-autodetection-20230507](https://cdash.dealii.org/build/1879) on site tester-tos
   - :disappointed: build warnings for configuration [GNU-13.1.1-master-minimal-20230116](https://cdash.dealii.org/build/1898) on site tester-tos
   - :disappointed: build warnings for configuration [GNU-13.1.1-master-standalone_kokkos-20230507](https://cdash.dealii.org/build/1878) on site tester-tos
   - :disappointed: build warnings for configuration [GNU-14.1.1-master-autodetection-20240704](https://cdash.dealii.org/build/1897) on site tester-tos
   - :angry: test failures for configuration [GNU-14.1.1-master-autodetection-20240704](https://cdash.dealii.org/build/1897) on site tester-tos
 - I will close and unpin this issue automatically once a full run is complete and compares cleanly to the baseline.
 - If closed I will reopen the issue if the testsuite run identified an additional regression compared to what I have reported so far.

[end automated regression testsuite report]

This looks like we're missing an include file somewhere:
```
/scratch/users/testsuite/testsuite-UB6aE0uC/dealii/tests/performance/timing_matrix_free_kokkos.cc:120:5: error: 'Portable' has not been declared
  120 |     Portable::FEEvaluation<dim, fe_degree, fe_degree + 1, 1, Number> *fe_eval,
      |     ^~~~~~~~
/scratch/users/testsuite/testsuite-UB6aE0uC/dealii/tests/performance/timing_matrix_free_kokkos.cc:120:70: error: expected unqualified-id before '*' token
  120 |     Portable::FEEvaluation<dim, fe_degree, fe_degree + 1, 1, Number> *fe_eval,
      |                                                                      ^
/scratch/users/testsuite/testsuite-UB6aE0uC/dealii/tests/performance/timing_matrix_free_kokkos.cc:120:69: error: expected ')' before '*' token
  120 |     Portable::FEEvaluation<dim, fe_degree, fe_degree + 1, 1, Number> *fe_eval,
      |                                                                     ^~
      |                                                                     )
/scratch/users/testsuite/testsuite-UB6aE0uC/dealii/tests/performance/timing_matrix_free_kokkos.cc:119:13: note: to match this '('
  119 |   operator()(
      |             ^
/scratch/users/testsuite/testsuite-UB6aE0uC/dealii/tests/performance/timing_matrix_free_kokkos.cc:120:15: error: expected ';' at end of member declaration
  120 |     Portable::FEEvaluation<dim, fe_degree, fe_degree + 1, 1, Number> *fe_eval,
      |               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                                                                     ;
/scratch/users/testsuite/testsuite-UB6aE0uC/dealii/tests/performance/timing_matrix_free_kokkos.cc:120:71: error: ISO C++ forbids declaration of 'fe_eval' with no type [-fpermissive]
  120 |     Portable::FEEvaluation<dim, fe_degree, fe_degree + 1, 1, Number> *fe_eval,
      |                                                                       ^~~~~~~
/scratch/users/testsuite/testsuite-UB6aE0uC/dealii/tests/performance/timing_matrix_free_kokkos.cc:121:5: error: expected unqualified-id before 'const'
  121 |     const int q_point) const
      |     ^~~~~
/scratch/users/testsuite/testsuite-UB6aE0uC/dealii/tests/performance/timing_matrix_free_kokkos.cc:133:29: error: 'Portable' has not been declared
  133 |              const typename Portable::MatrixFree<dim, Number>::Data *gpu_data,
      |                             ^~~~~~~~
/scratch/users/testsuite/testsuite-UB6aE0uC/dealii/tests/performance/timing_matrix_free_kokkos.cc:133:49: error: expected ',' or '...' before '<' token
  133 |              const typename Portable::MatrixFree<dim, Number>::Data *gpu_data,
      |                                                 ^
/scratch/users/testsuite/testsuite-UB6aE0uC/dealii/tests/performance/timing_matrix_free_kokkos.cc: In member function 'void LaplaceOperatorLocal<dim, fe_degree, Number>::operator()(unsigned int, int) const':
/scratch/users/testsuite/testsuite-UB6aE0uC/dealii/tests/performance/timing_matrix_free_kokkos.cc:140:5: error: 'Portable' has not been declared
  140 |     Portable::FEEvaluation<dim, fe_degree, fe_degree + 1, 1, Number> fe_eval(
      |     ^~~~~~~~
```

--- a/tests/performance/timing_matrix_free_kokkos.cc
+++ b/tests/performance/timing_matrix_free_kokkos.cc
@@ -37,6 +37,8 @@
 
 #include <deal.II/matrix_free/fe_evaluation.h>
 #include <deal.II/matrix_free/matrix_free.h>
+#include <deal.II/matrix_free/portable_fe_evaluation.h>
+#include <deal.II/matrix_free/portable_matrix_free.h>
 
 #include <deal.II/numerics/vector_tools.h>
 

